compiler/rustc_target/src/spec/mod.rs

```diff
@@ -932,6 +933,10 @@ supported_targets! {
     ("powerpc-wrs-vxworks-spe", powerpc_wrs_vxworks_spe),
     ("powerpc64-wrs-vxworks", powerpc64_wrs_vxworks),
 
+    ("aarch64-kmc-solid-asp3", aarch64_kmc_solid_asp3),
+    ("armv7a-kmc-solid-asp3-eabi", armv7a_kmc_solid_asp3_eabi),
+    ("armv7a-kmc-solid-asp3-eabihf", armv7a_kmc_solid_asp3_eabihf),
```

Comment:
I assumed these would always be 3 or 4 segments, but these look like 5, which might break tooling and confuse people. Would if be possible to refer to these as `solid_asp3` to avoid this?

Reply:
Applied the change in <https://github.com/kawadakk/rust/commit/b658351c1de2e5273f8881488a9a5e733be5b79b>, <https://github.com/kawadakk/libc/commit/26afb61a86e1d44ecbc7d47b15b328178a925052>, and <https://github.com/kawadakk/cc-rs/commit/e5411dbe6a39282f6b1a7881a7120ee3361218fe>.

I avoided using underscores because I had thought no existing target names have one. Turns out, there's one: `aarch64-unknown-linux-gnu_ilp32`